2023.2.0 (#26049)
home-assistant/home-assistant.github.io: source/_posts/2023-02-01-release-20232.markdown

- [Need help? Join the community!](#need-help-join-the-community)
- [Breaking Changes](#breaking-changes)
- [Farewell to the following](#farewell-to-the-following)
- [All changes](#all-changes)

Don't forget to [join our release party live stream on YouTube](https://www.youtube.com/watch?v=S3XRVsqRX9k) 1 February 2023, at 12:00 PST / 21:00 CET!

<lite-youtube videoid="S3XRVsqRX9k" videotitle="Home Assistant 2023.2 Release Party"></lite-youtube>

## Meet: Assist!

It's [Home Assistant's Year of the Voice](/blog/2022/12/20/year-of-voice/), and
it's our goal to let users control Home Assistant in their own language.
Together with the Home Assistant community, we have been hard at work building
the foundation for this.

Today we like you to meet:

<img class="no-shadow" src='/images/blog/2023-02/assist.png' alt='Screenshot showing the new assist feature being used to turn on lights'>

<img src='/images/assist/assist-icon.svg' alt='Assist icon' style='height: 32px' class='no-shadow'> Assist
allows you to use natural language to control Home Assistant. It is powered by
[Hassil](https://github.com/home-assistant/hassil) and the sentences
(contributed by 112! people) from the

> [Home Assistant Intent](https://home-assistant.github.io/intents/) project.

Reviewer: ```suggestion
[Home Assistant Intents](https://home-assistant.github.io/intents/) project.
```

We believe technology is meant to be played with, and projects should be
usable as soon as possible. Together with the community, we can then iterate
and refine. That's why today, we're delivering a basic experience supporting
a stunning 22! languages that work with Assist.

Go ahead, it is enabled by default; just tap the new Assist icon
<img src='/images/assist/assist-icon.svg' alt='Assist icon' style='height: 32px' class='no-shadow'>
at the top right of your dashboard to start using it.

Oh, and we are also releasing some fun stuff we've cooked up along the way!
[Read more about Assist](/docs/assist/) and other released voice features in the
[Chapter 1: Assist](/blog/2023/01/26/year-of-the-voice-chapter-1/) blogpost
and a [video presentation (including live demos) on YouTube](https://www.youtube.com/live/ixgNT3RETPg).

<lite-youtube videoid="ixgNT3RETPg" videotitle="Year of the Voice - Chapter 1"></lite-youtube>

## Streaming history

History cards now update live! We now stream the state changes while they are
happening and update the graphs in real-time 😲

<p class='img'>

> <img src='/images/blog/2023-02/streaming-history-cards.gif' alt='Screenshot showing a history card presenting the power usage of a kitching coffeemaker, updating live while brewing coffee'></a>

Reviewer: ```suggestion
<img src='/images/blog/2023-02/streaming-history-cards.gif' alt='Screenshot showing a history card presenting the power usage of a kitchen coffeemaker, updating live while brewing coffee'></a>
```

These history cards now show live power usage, while I'm brewing a cup of coffee.
</p>

But this isn't just limited to history cards, your
{% my history title="history dashboard" %} is now live too!

As a result of this all, we can now write less frequently to the database in our
recorder. It took years of changes to get us to this point (thanks [@bdraco] 🙏).
It greatly reduces resource usage and increases storage lifetimes significantly.
The previously introduced live logbook and now the live history were the
last pieces that made this possible.

As a side-effect, it should also resolve the authentication errors some people
experienced re-opening the iOS companion app when using history cards.

## Grouping sensors

The [group helper](/integrations/group) now supports sensors too! Sensor groups
can combine entities with a numeric state, like [sensors](/integrations/sensor),
[number](/integrations/number), and [number helpers](/integrations/input_number)
entities.

> <img class="no-shadow" src='/images/blog/2023-02/sensor-groups.png' alt='Screenshot the process of adding a new sensor group'>

Reviewer: ```suggestion
<img class="no-shadow" src='/images/blog/2023-02/sensor-groups.png' alt='Screenshot showing the process of adding a new sensor group'>
```

When you set up a new device in ESPHome and next add it to Home Assistant,
they will be able to exchange encryption keys. Removing the need for you to
find, copy & paste those. Just a single click and the setup is done! 🚀

But that is not all. What is really mind-blowing, is that the ESPHome
integration will now provide update entities when using the add-on!

<img class="no-shadow" src='/images/blog/2023-02/esphome-update-entities.png' alt='Screenshots, left one showing pending updates for ESPHome devices. Right one: an ESPHome device installing an update straight from Home Assistant'>

Right now, the update entities will only indicate if a device needs updating.
However, when the next major release of ESPHome (2023.2) becomes available,
it will support updating them too! Yes, at that point, you can update your
ESPHome devices directly from Home Assistant and even automate it!

Also, ESPHome is deprecating the old password-based authentication for its
API in favor of the more secure encryption key. If one of your ESPHome devices
is still using a plain password, Home Assistant will notify you by creating
an issue in your {% my repairs title="Repairs dashboard" %}.

> <img class="no-shadow" src='/images/blog/2023-02/esphome-repairs-encryption-key.png' alt='Screenshot showing an raised repair issue for a ESPHome device still using the deprecated API password instead of an API encryption key'>

Reviewer: ```suggestion
<img class="no-shadow" src='/images/blog/2023-02/esphome-repairs-encryption-key.png' alt='Screenshot showing a raised repair issue for an ESPHome device still using the deprecated API password instead of an API encryption key'>
```

- You can now add aliases to your areas too! Our new Assist feature
  leverages this!
- Got an energy sensor in `Wh`, but you'd like it to be `kWh`? You can now
  change the units of energy sensors in the entity settings.
- Each entity in the frontend has its own color, which is based on the current
  entity state. We now have [officially added support](/integrations/frontend/#state-color)
  for themes to customize these.
- The most awesome integration of all, got even better. [@Lash-L] added support
  for the battery state of your [Oral-B] toothbrush!
- The automations & scripts UI now support selecting other entities for
  above/below in the numeric state conditions & triggers. Thanks, [@karwosts]!
- The [Reolink] integration now supports FLV streams, cameras can be discovered
  on the network, but more importantly: binary sensors! For things
  like motion, person, vehicle detection, and... Doorbell presses!
  Thanks, [@starkillerOG]!
- There is now a [service] that allows for the creation of calendar events. So,
  you can now automate events to be added to your calendars! Amazing, [@allenporter]!
- When selecting an entity from a list in the UI, it will now exclude already

>   added entities. Helpful! Thanks, [@piitaya]!

Reviewer: Consider adding some context about why the entities are selected explaining why the exclusion makes sense. Not a big deal though.

Author: It has many use cases, but making it clear is hard I guess. Tried wording it differently by describing the annoying experience many of us are familiar with.

suffix. The accumulated sensors have been introduced to support net energy
export/import metering.

Newer versions of IoTaWatt provide "Integrators", which allow achieving the
same more efficiently and with higher accuracy.

You should configure [Integrators](https://docs.iotawatt.com/en/master/integrators.html)
to calculate the net energy export and import.

([@agners] - [#86611]) ([documentation](/integrations/iotawatt))

[@agners]: https://github.com/agners
[#86611]: https://github.com/home-assistant/core/pull/86611

{% enddetails %}

{% details "Matter" %}

The Matter integration now requires running Matter Server version `2.0.0`.

If you're using the Matter Server add-on to run the Matter Server,
you will need version `3.0.0` of the add-on.

We had to change our vendor ID in the add-on; as a result, all devices
need to be recommissioned.

([@MartinHjelmare] - [#86470]) ([documentation](/integrations/matter))

[@MartinHjelmare]: https://github.com/MartinHjelmare
[#86470]: https://github.com/home-assistant/core/pull/86470

{% enddetails %}

{% details "Nettigo Air Monitor" %}

The states of air quality sensors have been standardized to match Home Assistant

> core rules. This affects two states some sensors of NAM can provide:

Reviewer: ```suggestion
core rules. This affects two states of the sensors that NAM can provide:
```

have been deprecated. Any existing automations or scripts that make service
calls to `rainbird.set_rain_delay` will need to be replaced with a call
to `number.set_value`. Additionally, the existing service call was updated to
require an integration entry configuration since it previously had undefined
behavior, so it is recommended to move directly to the new number service call.

([@allenporter] - [#85271] [#86208]) ([documentation](/integrations/rainbird))

[@allenporter]: https://github.com/allenporter
[#85271]: https://github.com/home-assistant/core/pull/85271
[#86208]: https://github.com/home-assistant/core/pull/86208

{% enddetails %}

{% details "Recorder" %}

The recorder's default commit interval has been increased to 5 seconds to
reduce disk I/O and increase storage longevity.

As the history and logbook integrations both support live streaming, which
avoids the need to read the database after the initial sync, there is no
remaining core functionality that needs a lower commit interval.

The [`commit_interval`](/integrations/recorder/#commit_interval) can be adjusted
to a lower value in the event an integration has not been updated to watch for

> events instead of reading the database needs to read the database sooner.

Reviewer: ```suggestion
events instead of reading the database and needs to read the database sooner.
```

The options flow has been removed from the integration, as the only
configurable setting `ignored_sources` was no longer relevant
and incompatible with this change.

([@Drafteed] - [#85288]) ([documentation](/integrations/braviatv))

[@Drafteed]: https://github.com/Drafteed
[#85288]: https://github.com/home-assistant/core/pull/85288

{% enddetails %}

{% details "Spain electricity hourly pricing (PVPC)" %}

The previously deprecated YAML configuration of the Spain electricity hourly
pricing (PVPC) integration has been removed.

Spain electricity hourly pricing (PVPC) is now configured via the UI. Any
existing YAML configuration has been imported in previous releases and can now
be safely removed from your YAML configuration files.

([@azogue] - [#85614]) ([documentation](/integrations/pvpc_hourly_pricing))

[@azogue]: https://github.com/azogue
[#85614]: https://github.com/home-assistant/core/pull/85614

{% enddetails %}

{% details "Tado" %}

The swing modes of Tado have changed. This is to standardize with other
integrations, and so it can support translations.

- `ON` became the lowercase `on`
- `OFF` became the lowercase `off`

> If you used these swing modes in your automation or scripts, you might

Reviewer: ```suggestion
If you used these swing modes in your automations or scripts, you might
```
